kairon/events/definitions/upload_handler.py

```
@@ -0,0 +1,57 @@
+
+        payload=self.upload_handler.create_payload(**kwargs)
+        UploadHandlerLogProcessor.add_log(bot=self.upload_handler.bot, user=self.upload_handler.user, event_status=EVENT_STATUS.ENQUEUED.value)
+        try:
+            Utility.request_event_server(EventClass.upload_file_handler, payload)
+        except Exception as e:
+            logger.error(str(e))
+            UploadHandlerLogProcessor.add_log(bot=self.upload_handler.bot, user=self.upload_handler.user, exception=str(e),
+                                              event_status=EVENT_STATUS.FAIL.value)
+            content_dir = os.path.join('file_content_upload_records', self.upload_handler.bot)
+            Utility.delete_directory(content_dir, True)
+            raise e
```

Ruff (0.12.2)

Use `raise` without specifying exception name

Remove exception name

(TRY201)